virttest/qemu_vm.py

```diff
@@ -1945,6 +1960,18 @@ def add_secure_guest_descriptor(params):
         # ARM target support SMP 'clusters' parameter since qemu 7.0
         vcpu_clusters = int(vcpu_clusters)
 
+        if not enable_drawers:
+            # Set drawers=1 when computing missing values
+            vcpu_drawers = 1
```

Comment:
Could these conditionals about vcpu drawers and books be removed when setting the default value at creation?
```
        vcpu_drawers = int(params.get("vcpu_drawers", 1))
        enable_drawers = vcpu_drawers != "INVALID" and Flags.SMP_DRAWERS in devices.caps
```

Reply:
yes, vt will give the default value of 1